packages/inferno-from-react-generator/src/__tests__/test-cases/import-without-react.jsx

```diff
@@ -0,0 +1,7 @@
+import React, { useEffect, useMemo } from 'react';
+//* Component={"name":"DxComponent"}
+function DxComponent() {
+    const x = useEffect.toString() + useMemo.toString();
```

Comment:
Minor comment

Let's name it ```className```, but in test, it's not important

Reply:
done